packages/tldraw/src/lib/shapes/shared/freehand/getStrokeOutlinePoints.ts

```diff
@@ -7,26 +7,17 @@ const { PI } = Math
 const FIXED_PI = PI + 0.0001
 
 /**
- * ## getStrokeOutlinePoints
- *
- * Get an array of points (as `[x, y]`) representing the outline of a stroke.
- *
- * @param points - An array of StrokePoints as returned from `getStrokePoints`.
- * @param options - An object with options.
- * @public
+ * @internal
  */
-export function getStrokeOutlinePoints(
+export function getStrokeOutlineTracks(
 	strokePoints: StrokePoint[],
 	options: StrokeOptions = {}
-): Vec2d[] {
-	const { size = 16, smoothing = 0.5, start = {}, end = {}, last: isComplete = false } = options
-
-	const { cap: capStart = true } = start
-	const { cap: capEnd = true } = end
+): { left: Vec2d[]; right: Vec2d[] } {
```

Comment:
To allow me to draw the caps with SVG arcs, I extracted out the part of this that gets the left and right tracks so I could use it on its own.